utilities/cloudharness_utilities/deployment-configuration/helm/templates/auto-database-mongo.yaml

```diff
@@ -0,0 +1,26 @@
+{{- define "deploy_utils.database.mongo" }}
+        image: {{ .app.harness.database.mongo.image }}
+        env:
+          - name: MONGO_INITDB_ROOT_USERNAME
+            value: {{ .app.harness.database.user | quote }}
+          - name: MONGO_INITDB_ROOT_PASSWORD
```

Comment:
when we change the password we will have a problem
I know this is a general issue and we need to solve it in a generic way
I was thinking of:
* use of secrets
* use of configmap

what do you think and do you have other ideas?

Reply:
I would go with secrets to store any kind of confidential data, it's as far as I know what K8s recommends and has already some nice functionality and support for various secret types, used it in previous projects and found it to be a good choice